pandas/tests/scalar/timestamp/test_arithmetic.py

```diff
@@ -16,6 +16,10 @@
 
 import pandas._testing as tm
 
+pytestmark = pytest.mark.filterwarnings(
+    "ignore:Timestamp.freq is deprecated:FutureWarning"
```

Comment:
Can you apply this mark to only the tests that need it? (I would expect not all tests to need it?)

Reply:
good idea, will do